### ISSUE TITLE
Phonegap: sms is not defined

### ISSUE THREAD

**Issue by vuphuctho** [cordova-sms/cordova-sms-plugin#67]
Hi, I received "sms is not defined" when using the plugin. This is what I have done so far:
1. `phonegap create sms` and go inside created project
2. `cordova plugin add cordova-sms-plugin`
3. `cordova platform add android`
4. Add the sample HTML and Javascript code

In this [post](https://github.com/aharris88/phonegap-sms-plugin/issues/12), there supposes to be a java file in "platforms/android/src/org/apache/cordova/plugin/sms/Sms.java", but it isn't. I tried to remove and add the plugin again, but it doesn't work.

**Comment by dbaq** [comment]
hey @vuphuctho,

The plugin is not installed properly. Take a look at http://docs.phonegap.com/en/edge/guide_cli_index.md.html#The%20Command-Line%20Interface

**Comment by dbaq** [comment]
Also please answer the following questions: https://github.com/cordova-sms/cordova-sms-plugin/issues/48#issuecomment-134241054

**Comment by vuphuctho** [comment]
Hi @dbaq 
I just figured out the reason. Since phonegap-cli and Phonegap Desktop does not allow third-party plugins, sms will not be added if I use it using command "phonegap serve" or run with Phonegap Desktop. Everything works fine when compiling the app using [Phonegap Build](https://build.phonegap.com/). 
Please correct me if I am wrong.

**Comment by dbaq** [comment]
Great you figured this out. IMHO, you should use cordova directly (or ionic-cli if you are using ionic).

**Comment by muli-cohen** [comment]
Hi, Thanks for this great plugin, but I also got this issue. I installed the plugin via both cli and phonegap, but when I try to send an sms with my device nothing happens. The intersting thing is, that during installation of the apk, I get a notion that the app has access to "send SMS messages - this may cost you money". Also this plugin is showing on the phongap build plugin section. What am I doing wrong?
I'm very new to PhoneGap\Cordova.

Thanks again,

Samuel.
